### ISSUE TITLE
Inventory items should leave the inventory when equipped

### ISSUE THREAD

**Issue by ASteinheiser** [ASteinheiser/react-rpg.com#25]
This should be pretty easy to implement, just have to be careful on the un-equip.
Need to make sure we have inventory space before taking off item.

**Comment by ASteinheiser** [comment]
Also, if this change is made, we should increase the backpack upgrade price. I lowered it from 100 -> 50 gold because it was basically an essential upgrade. Needs to be back to 100.

**Comment by Alphanumber** [comment]
Would the backpack upgrade even be necessary after the change?

**Comment by ASteinheiser** [comment]
Good point!! You are so right.
We should just remove the backpack upgrade when this change is made.

**Comment by ASteinheiser** [comment]
@Alphanumber The more I think about it, I feel like maybe we could leave the backpack upgrade but make it more expensive. This would allow us to introduce other consumable items later on.

What do you think?

**Comment by Alphanumber** [comment]
Sure, that sounds reasonable if new consumables would eventually be introduced, the new consumables are affordable, and inventory space is scarce enough. Right now, with only potions, the backpack upgrade would be a bit redundant after equipment is taken out of the inventory into equipment slots.

Did you have anything in mind as far as new consumables go?

**Comment by ASteinheiser** [comment]
Good point. I think I will go through with disabling the inventory until we have a reason to expand it.

For new consumables, they would probably be a bit down the road, but something like arrow stacks for an archer class or mana potions for a mage class.
Or maybe there are special artifacts or potions that could give you temporary stat boosts.
